src/modules/analytics/AnalyticsEvent.js

```diff
@@ -1,20 +1,31 @@
 // @flow
 
+export type noop = (...args: any) => {};
+export type FireAnalyticsEvent = (event: AnalyticsEvent, channel: string) => void | noop;
```

Comment:
this reads to me like it returns `void` or `noop` - are you sure that this syntax is correct?